src/olympia/hero/models.py

```diff
@@ -157,7 +157,10 @@ def __str__(self):
 
     @property
     def image_url(self):
-        return f'{FEATURED_IMAGE_URL}{self.image}' if self.image else None
+        if self.select_image:
+            return f'{self.select_image.custom_image.url}'
+        else:
+            None
```

Comment:
This code is a little weird - I guess you meant to write `return None`? 
(You didn't like the `return xxx if yyy else zzz` style?)

Reply:
No I like the style! flake8 was giving me problems about the line being too long. I'll revert to the original style and add parenthesis.